src/index.js

```diff
@@ -119,8 +120,18 @@ function(err, results) {
     shell.exec('npm version --no-git-tag-version ' + oldVersion);
   }
 
-  // ### STEP 8 - Run if any pre commit script has been specified (DESTRUCTIVE OPERATION)
+  // ### STEP 8 - Run if any pre commit script has been specified (DESTRUCTIVE OPERATION?)
   lib.runPreCommitScript(program.preCommit, version);
+
   // ### STEP 9 - Tag and push (DESTRUCTIVE OPERATION)
-  if (!program.dryrun) lib.addFilesAndCreateTag(version);
+  let pushResultCode = 1; // Default to failure signal
+
+  if (!program.dryrun) {
+    pushResultCode = lib.addFilesAndCreateTag(version);
+  }
+
+  // ### STEP 10 - Run after successful push (DESTRUCTIVE OPERATION?)
+  if (pushResultCode === 0 || program.dryrun) {
```

Comment:
I can change that. The problem is I can't figure out how to test that the script is called unless it executes in dry-run. When running the test normally, the result code from `git push ...` is 128, because we haven't setup a remote server. Previous to this feature, nothing was checking for the return code so it didn't matter. Now we only want to run a script when the exit code is zero. Suggestions?

Also, I'm going to refactor the preCommit & postSuccess code as it is identical and should be a single function.